### PR TITLE
chore(bridges): increase log level for bridge buffer overflow reports

### PR DESCRIPTION
**Pull request by savonarola**
Fixes [EMQX-12458](https://emqx.atlassian.net/browse/EMQX-12458)

Release version: e/v5.8.0

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [na] Added tests for the changes
- [na] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [x] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [x] Schema changes are backward compatible

[EMQX-12458]: https://emqx.atlassian.net/browse/EMQX-12458?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/emqx_resource/src/emqx_resource_buffer_worker.erl [hunk 1]
```diff
@@ -1777,7 +1777,7 @@ append_queue(Id, Index, Q, Queries) ->
                 ok = replayq:ack(Q1, QAckRef),
                 Dropped = length(Items2),
                 Counters = #{dropped_queue_full => Dropped},
-                ?SLOG(info, #{
+                ?SLOG(warning, #{
```
[the remark this answers is not in the document]
Comment (zmstone) [reply apps/emqx_resource/src/emqx_resource_buffer_worker.erl:1780]: good point, maybe add a new THROTTLE instead.

[the remark this answers is not in the document]
Comment (savonarola) [reply apps/emqx_resource/src/emqx_resource_buffer_worker.erl]: ✅

File: apps/emqx_resource/src/emqx_resource_buffer_worker.erl [hunk 2]
```diff
@@ -1777,7 +1777,7 @@ append_queue(Id, Index, Q, Queries) ->
                 ok = replayq:ack(Q1, QAckRef),
                 Dropped = length(Items2),
                 Counters = #{dropped_queue_full => Dropped},
-                ?SLOG(info, #{
+                ?SLOG_THROTTLE(warning, #{
                     msg => "buffer_worker_overflow",
```
Comment (zmstone) [inline apps/emqx_resource/src/emqx_resource_buffer_worker.erl:1781]: the throttle ID must be an atom and enumerated in the schema

e.g. `apps/emqx_conf/src/emqx_conf_schema.erl:84:5:    cannot_publish_to_topic_due_to_not_authorized,`

Comment (savonarola) [reply apps/emqx_resource/src/emqx_resource_buffer_worker.erl]: ✅ A bit counterintuitive 😅

File: apps/emqx_conf/src/emqx_conf_schema.erl [hunk 3]
```diff
@@ -85,7 +85,8 @@
     cannot_publish_to_topic_due_to_quota_exceeded,
     connection_rejected_due_to_license_limit_reached,
     dropped_msg_due_to_mqueue_is_full,
-    socket_receive_paused_by_rate_limit
+    socket_receive_paused_by_rate_limit,
+    buffer_worker_overflow
```
Comment (zmstone) [inline apps/emqx_conf/src/emqx_conf_schema.erl:89]: data_integration_bridge_buffer_overflow

File: changes/ce/feat-13144.en.md [hunk 4]
```diff
@@ -0,0 +1 @@
+Emit a `warning` log event when bridge buffers are overflown and messages get dropped. Previously, the log events had `info` level and were not visible with the default log level.
```
Comment (zmstone) [inline changes/ce/feat-13144.en.md:1]:
```suggestion
Emit a `warning` (throttled) log when bridge buffers are overflown and messages get dropped. Previously, the log events had `info` level and were not visible with the default log level.
```

**Review by zmstone — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-3354)

**Pull request merged**